mk/tools.mk

```diff
@@ -17,39 +17,37 @@
 ##############################
 
 # Set up ARM (STM32) SDK
-ARM_SDK_BASE_DIR ?= $(TOOLS_DIR)/arm-gnu-toolchain-13.3.rel1
 # Checked below, Should match the output of $(shell arm-none-eabi-gcc -dumpversion)
+# must match arm-none-eabi-gcc-<version> file in arm sdk distribution
 GCC_REQUIRED_VERSION ?= 13.3.1
 
 ## arm_sdk_install   : Install Arm SDK
 .PHONY: arm_sdk_install
 
 # source: https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads
-ifeq ($(OSFAMILY), linux)
+ifeq ($(OSFAMILY)-$(ARCHFAMILY), linux-x86_64)
   ARM_SDK_URL := https://developer.arm.com/-/media/Files/downloads/gnu/13.3.rel1/binrel/arm-gnu-toolchain-13.3.rel1-x86_64-arm-none-eabi.tar.xz
-  ARM_SDK_DIR := $(ARM_SDK_BASE_DIR)-x86_64-arm-none-eabi
-endif
-
-ifeq ($(OSFAMILY), macosx)
-  # Check for Apple Silicon
-  UNAME_PROCESSOR := $(shell uname -p)
-  ifeq ($(UNAME_PROCESSOR), arm)
-    ARM_SDK_URL := https://developer.arm.com/-/media/Files/downloads/gnu/13.3.rel1/binrel/arm-gnu-toolchain-13.3.rel1-darwin-arm64-arm-none-eabi.tar.xz
-    ARM_SDK_DIR := $(ARM_SDK_BASE_DIR)-darwin-arm64-arm-none-eabi
-  else
-    ARM_SDK_URL := https://developer.arm.com/-/media/Files/downloads/gnu/13.3.rel1/binrel/arm-gnu-toolchain-13.3.rel1-darwin-x86_64-arm-none-eabi.tar.xz
-    ARM_SDK_DIR := $(ARM_SDK_BASE_DIR)-darwin-x86_64-arm-none-eabi
-  endif
-endif
-
-ifeq ($(OSFAMILY), windows)
+  DL_CHECKSUM = 0601a9588bc5b9c99ad2b56133b7f118
```

Comment:
Would be nice if we could extract `0601a9588bc5b9c99ad2b56133b7f118` from the [asc file](https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu/13.3.rel1/binrel/arm-gnu-toolchain-13.3.rel1-x86_64-arm-none-eabi.tar.xz.asc) :)